### DISCUSSION TITLE
text block - special characters

### DISCUSSION THREAD

**Discussion by eswanzey** [vanjarosoftware/Vanjaro.Platform#1056] [Q&A]
It seems to me that there isn't a way to enter special characters in a text block unless they are clipboarded (pasted) in. Is that true?

**Comment by meetmandeep** [comment]
I don't have any problems adding any special characters in text block. Can you provide more information?

**Reply by eswanzey** [reply]
I can right-click the field and get my browser to give me a visual pick list. I'm not seeing an icon in the UI for doing it natively in the Vanjaro editor.

**Reply by meetmandeep** [reply]
Can you add a screenshot?

**Reply by eswanzey** [reply]
I was expecting to see it in the left editing menu below -

![2021-04-14_15-09-37](https://user-images.githubusercontent.com/8103212/114766395-597a9c80-9d34-11eb-850e-01f2c64e767e.jpg)

**Reply by meetmandeep** [reply]
> I can right-click the field and get my browser to give me a visual pick list.

I'm still lost. Can you provide a screenshot of what you see in your native browser?

**Reply by eswanzey** [reply]
I had someone ask me about this today, and remembered this thread. Here's what we miss from CKeditor - the ability to set sub- and superscript, and the ability to insert special characters. Is that outside the bounds of what the text block is meant for? Maybe the only way is to insert an HTML block?

![2021-11-17_13-40-37](https://user-images.githubusercontent.com/8103212/142263146-2ed684ce-d41f-45e0-bff4-61fac59e7ed9.jpg)
.

**Reply by meetmandeep** [reply]
I suppose those can be added to the inline editor.
